src/testhelpers.rs

```diff
@@ -57,10 +54,10 @@ impl Default for TestCounterEntity {
 }
 
 impl<'a> Aggregator<TestEvents, String, PgQuery<'a>> for TestCounterEntity {
-    fn apply_event(acc: Self, event: &TestEvents) -> Self {
-        let counter = match event {
-            TestEvents::Inc(inc) => acc.counter + inc.by,
-            TestEvents::Dec(dec) => acc.counter - dec.by,
+    fn apply_event(acc: Self, event: &Event<TestEvents>) -> Self {
+        let counter = match event.data {
+            TestEvents::Inc(ref inc) => acc.counter + inc.by,
```

Comment:
I will never get used to this weird `ref` syntax only used on match clauses.

![wtf](https://media.giphy.com/media/guM0lbsBTYtBC/giphy-downsized.gif)